shopfloor_reception_mobile/static/src/scenario/reception_states.js

```diff
@@ -0,0 +1,295 @@
+/**
+ * Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
+ * Copyright 2023 Michael Tietz (MT Software) <mtietz@mt-software.de>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+ */
+
+/*
+    Define states for reception scenario.
+    @param $instance VueJS component instance
+*/
+export const reception_states = function ($instance) {
+    return {
+        init: {
+            enter: () => {
+                this.wait_call(this.odoo.call("start"));
```

Comment:
if you don't bind `this` to `$instance` you have to replace all this `this` w/ `$instance`.

Reply:
oh yes i forget to replace it